index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/98db508811.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="background-n">
+
+        <div class="background-opacity">
+            <div class="container">
+                <nav>
+                    <form action="#" method="POST">
+                        <input type="text" name="city" placeholder="Введите название города">
+                        <button> click</button>
+                    </form>
+                    <div class="city-contry">
+                        <p> BELARUS,</p>
+                        <p>MINSK</p>
+                    </div>
+
+                </nav>
+                <main>
+                    <div class="today_weather">
+                        <div class="today">
+                            <p class="today_data-name">Today</p>
+                            <p class="today_data">15/09/2021</p>
+                            <p class="today_day-of-week">Wednesday</p>
+                        </div>
+                        <div class="today_weather-weather">
+                            <i class="fas fa-sun today_i"></i>
+                            <p class="today_weather-day">°C</p>
+                            <i class="fas fa-cloud-moon today_i"></i>
+                            <p class="today_weather-night">°C</p>
+                            <p class="today_weather-pressure">Pressure:</p>
+                            <p class="today_weather-wind">Wind:</p>
+                            <p class="today_weather-humidity">Humidity:</p>
+                        </div>
+                    </div>
```

Comment:
On next task pls remake this into some sort of component
Create it with JS, because this elements will change

If you will create them with JS it will be much easier to change data

For example right now you need to hardcode all this 9000 classes, then find all this elements with `Document.querySelector()` and only after somehow put some values. And values also will be 50% hardcoded, because there are keywords like **Pressure**, **Wind** etc.